src/include/traffic_cop/traffic_cop_util.h

```diff
@@ -7,6 +7,8 @@
 #include "common/managed_pointer.h"
 #include "network/network_defs.h"
 #include "optimizer/optimize_result.h"
+#include "optimizer/property_set.h"
```

Comment:
forward declare this

Reply:
Done.... and now I know what forward declare means...

Reply:
generally it is something worth doing in commonly included .h files that could result in a huge include blowup